### PR TITLE
[ADD] 8.0 l10n_be_eco_tax

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

File: l10n_be_eco_tax/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+There is no specific installation procedure for this module.
+
+Configuration
+=============
+
+If you install this module after instanciating the chart of account
+for your company, it is recommanded to use the account_chart_update
+module from the OCA/account-financial-tools repository .
+
+You may also need to constomize your invoice layout to render
```
Comment (alexis-via) [inline l10n_be_eco_tax/README.rst]: constomize -> customize

File: l10n_be_eco_tax/README.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+these eco taxes in the desired way.
+
+Usage
+=====
+
+To use this module, you need to configure add the BEBAT and/or RECUPEL
```
Comment (alexis-via) [inline l10n_be_eco_tax/README.rst]: remove the word "configure"

**Comment by gurneyalex** [comment]
:+1:

**Comment by jbaudoux** [comment]
@sbidoul Thanks for the merge of the port to 9.0. I suggest to also merge this PR for 8.0.

**Comment by sbidoul** [comment]
Indeed. Done.

**Pull request merged**